src/algebra/group_with_zero/basic.lean

```diff
@@ -120,7 +120,7 @@ are nonzero. -/
 theorem mul_ne_zero_iff : a * b ≠ 0 ↔ a ≠ 0 ∧ b ≠ 0 :=
 (not_congr mul_eq_zero).trans not_or_distrib
 
-theorem mul_ne_zero (ha : a ≠ 0) (hb : b ≠ 0) : a * b ≠ 0 :=
+@[field_simps] theorem mul_ne_zero (ha : a ≠ 0) (hb : b ≠ 0) : a * b ≠ 0 :=
 mul_ne_zero_iff.2 ⟨ha, hb⟩
```

Comment:
Does this mean we can remove the attribute from here?

https://github.com/leanprover-community/mathlib/blob/461865b23259ca8a8f5b322ebfc6ecea854cd9b7/src/data/real/nnreal.lean#L435-L436

Reply:
> Does this mean we can remove the attribute (and also the lemma?) from here?
> 
> https://github.com/leanprover-community/mathlib/blob/461865b23259ca8a8f5b322ebfc6ecea854cd9b7/src/data/real/nnreal.lean#L435-L436

Reply:
Done

Reply:
Did you push this? I agree the whole lemma can probably go now.

Reply:
Oops, I had forgotten to push. Thanks for the reminder!